layouts/shortcodes/downloads.html

```diff
@@ -62,9 +63,10 @@
 
   {{ range $valid_version := $.Scratch.Get "latest_versions" }}
   {{ range $manifest.istio_distributions }}
-  {{ if and (findRE $valid_version .version) (ne $os_type "unspecified") (ne .flavor "tetratefips")}}
+  {{ if and (findRE $valid_version .version) (ne $os_type "unspecified") }}
   {{ $release_url := index (.release_notes) 0}}
   <tr>
+    <td> {{ .version }}-{{ .flavor }}-v{{ .flavor_version }} </td>
```

Comment:
Not sure how it gets flavor_version here? The URLs are broken has a double "0" e.g.  tetrate-v00

Reply:
Disregard, I think I found it.